README.md

```diff
@@ -6,6 +6,8 @@
 [![GoDoc](https://godoc.org/github.com/google/ko?status.svg)](https://godoc.org/github.com/google/ko)
 [![Go Report Card](https://goreportcard.com/badge/google/ko)](https://goreportcard.com/report/google/ko)
 
+<img src="https://github.com/google/ko/raw/master/logo/ko.png" width="300">
```

Comment:
Change this to `<img src="./logo/ko.png" width="300">`

Reply:
I was duplicating what is done with https://github.com/kubernetes/kubernetes

Reply:
That works fine.